package.json

```diff
@@ -40,12 +44,13 @@
     }
   },
   "dependencies": {
+    "autoprefixer-loader": "^2.0.0",
     "autoprefixer-core": "^5.2.0",
     "babel": "^5.8.21",
     "babel-loader": "^5.3.2",
     "babel-plugin-typecheck": "^1.2.0",
     "babel-runtime": "^5.8.20",
-    "basscss-sass": "^1.0.2",
+    "basscss-sass": "nicolasiensen/basscss-sass#v1.0.2-branch",
```

Comment:
In that case, let's merge this and later I can make a branch just to update it and ensure styles don't break. :dancers: